src/js/stores/font.js

```diff
@@ -91,6 +91,98 @@ define(function (require, exports, module) {
 
             return fontObj && fontObj.postScriptName;
         },
+
+        getTypeObjectFromLayer: function (layer) {
+            if (layer.kind !== layer.layerKinds.TEXT) {
+                return null;
+            }
+
+            var obj = {},
+                textStyle = layer.text.characterStyles.first(),
+                psName = textStyle.postScriptName,
+                fontObj = this._postScriptMap.get(psName, null);
+                
+            if (!fontObj) {
+                return null;
+            }
+            var family = fontObj.family,
+                fontRecord = this._familyMap.get(family),
+                psObj = fontRecord.get(fontObj.font, null);
+
+            if (!psObj) {
+                return null;
+            }
+
+            obj.adbeFont = {
+                family: fontObj.family,
+                name: fontObj.font,
+                postScriptName: psObj.postScriptName,
+                style: psObj.style
+            };
+
+            obj.fontFamily = fontObj.family;
+
+            if (textStyle.textSize) {
+                obj.fontSize = {
+                    "type": "pt",
+                    "value": textStyle.textSize
+                };
+            }
+
+            var style = psObj.style.toLowerCase();
```

Comment:
In this case, the object expected is different from what our renders expect. I suggest creating a code cleanup issue for this, instead of blocking the PR.